app/models/job/Job.scala

```diff
@@ -223,14 +254,16 @@ class JobService @Inject()(wkConf: WkConf,
     ()
   }
 
-  private def trackNewlySuccessful(jobBeforeChange: Job, newStatus: JobStatus): Unit = {
+  private def trackNewlySuccessful(jobBeforeChange: Job, jobAfterChange: Job): Unit = {
     for {
       user <- userDAO.findOne(jobBeforeChange._owner)(GlobalAccessContext)
       organization <- organizationDAO.findOne(user._organization)(GlobalAccessContext)
+      resultLink = jobAfterChange.resultLink(organization.name)
+      resultLinkMrkdwn = resultLink.map(l => s" <${wkConf.Http.uri}$l|Result Link>").getOrElse("")
```

Comment:
```suggestion
      resultLinkMrkdwn = resultLink.map(l => s" <${wkConf.Http.uri}$l|Result>").getOrElse("")
```

If that's the clickable link, I think "Link" can be omitted.